sources/packages/api/src/services/student/student.service.models.ts

```diff
@@ -8,6 +8,7 @@ import { AddressInfo } from "../../database/entities";
 export interface StudentInfo extends AddressInfo {
   phone: string;
   sinNumber: string;
+  sinConsent: boolean;
```

Comment:
This property was added but apparently, it is not used while saving the student.